tools/ci_build/github/windows/setup_env_eager.bat

```diff
@@ -0,0 +1,2 @@
+set PATH=C:\azcopy;C:\\hostedtoolcache\\windows\\Python\\3.7.9\\x64\\lib\\site-packages\\torch\\lib;%PATH%
```

Comment:
as we synced offline, we don't need this file, right?

Reply:
Let me test it if it still required.